prompttools/structures/prompt.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class PromptTest:
```

Comment:
See my other comment about using experiments for your test setup. These structures are unnecessary.